OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVREntryPoint.cpp

```diff
@@ -29,13 +29,33 @@ DEFINE_LOG_CATEGORY(OSVREntryPointLog);
 
 OSVREntryPoint::OSVREntryPoint()
 {
+}
+
+void OSVREntryPoint::Initialize()
+{
+    // avoid double initialization
+    if (osvrClientContext != nullptr)
+    {
+        // too spammy to log here
+        return;
+    }
+
+    // hack: to avoid delay on OSVR server connection attempt when not using OSVR, don't initialize if nohmd command line param specified
+    if (FParse::Param(FCommandLine::Get(), TEXT("nohmd")))

```

Comment:
OSVRInput may need the entry point even if nohmd is set. Can this skipping behavior be parameterized?